ga4gh/configtest.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import division
```

Comment:
Can we get a module docstring here please? Doesn't have to be anything elaborate, just a quick description of what the module is for.